ipfs/ipfsd.go

```diff
@@ -79,7 +79,11 @@ func StartIpfsDaemon(ctx context.Context, ipfsPath string) error {
 
 	if err := cmd.Wait(); err != nil {
 		// glog.Infof("%v", stderr)
-		glog.Errorf("Error: %v - %v", err, out)
+		if out.Len() < 500 {
```

Comment:
What's the idea behind the two different error messages - just for debugging?

Reply:
Yeah - ideally we want to print out the error, except for when the output is large (in which case it's just a video segment).  I've captured it in https://github.com/livepeer/go-livepeer/issues/159